src/main/java/it/gov/pagopa/pu/classification/repository/AssessmentsDetailRepository.java

```diff
@@ -0,0 +1,11 @@
+package it.gov.pagopa.pu.classification.repository;
+
+import it.gov.pagopa.pu.classification.model.AssessmentsDetail;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+
+
+@RepositoryRestResource(path = "assessments-detail")
```

Comment:
let's use plural form for repository paths
```suggestion
@RepositoryRestResource(path = "assessments-details")
```